src/Workspaces/Core/Portable/Simplification/AbstractSimplificationService.cs

```diff
@@ -178,89 +178,92 @@ protected virtual SyntaxNode TransformReducedNode(SyntaxNode reducedNode, Syntax
             {
                 var nodeOrTokenToReduce = nodesAndTokensToReduce[i];
                 simplifyTasks[i] = Task.Run(async () =>
-            {
-                var nodeOrToken = nodeOrTokenToReduce.OriginalNodeOrToken;
```

Comment:
All this code stays the same.  it just got indented.  I will point out what actually changed.